src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,205 @@
+package duke;
+
+// import javafx.scene.control.Label;
```

Comment:
I understand that you have moved these comments out to another directory, I think you can delete these comments to keep the file cleaner and shorter